heltour/tournament/models.py

```diff
@@ -1583,7 +1584,19 @@ def save(self, *args, **kwargs):
                 old_black_setting = PlayerNotificationSetting.get_or_default(
                     player_id=self.initial_black_id, type='before_game_time', league=league)
                 old_black_setting.save()
-
+                
+            #set players available if game gets scheduled and either of them is set unavailable,
+            #do not set a player with a red card available, though.
+            if not SeasonPlayer.objects.filter(Q(player__id=self.white_id) & Q(season=self.get_round().season) & Q(games_missed__gte=2)).exists():
+                PlayerAvailability.objects.filter(
+                    (Q(player__id=self.white_id) & Q(round=self.get_round())
```

Comment:
When using a `.filter()` specying multiple attributes to filter on already uses an `AND`. So that means that:
```python
.filter(Q(foo=X)&Q(bar=Y))
```
Is equivalent to:
```python
.filter(foo=X, bar=Y)
```

Reply:
You only need to use the `Q` objects for more complicated logic, like `OR` or mixing `OR` with `AND`